### ISSUE TITLE
[Bug] YTNoModernUI is somewhat broken on video pages

### ISSUE THREAD

**Issue by millionsofplayers** [arichornlover/uYouEnhanced#88]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Select uYou Label

- [ ] uYou

### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Device info

- iOS/iPadOS version: 17 PB3 (latest)
- Device model: iPhone 14 Pro 
- Sideload tool (AltStore, Sideloadly, TrollStore,...): AltStore
- The specific version of uYouPlus (**latest or newest is NOT a version number!**): 18.30.6-3.0.1

### Describe the bug

When I enable the NoModernUI option, the video page is somewhat broken. The like/dislike, title and subscribe parts have been changed and the corners are unrounded, but everything else is the same.

### Steps to reproduce the issue

1. enable YTNoModernUI
2. go to a video page
3. there's the bug

### Are you using the newest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

### Additional context

Here's a screenshot of the issue. Sorry if the issue isn't written well. [Screenshot](https://files.catbox.moe/ikwedf.PNG)

**Comment by millionsofplayers** [comment]
Also unrelated, but I can’t turn on notifications on the modded app. Is there a way to make the allow notifications pop up happen again?

**Comment by millionsofplayers** [comment]
Does the spoof thing break anything or does it just make the video page better with that option?

**Comment by Felting** [comment]
The old dark theme option doesn't work on video options with nomodernui and fake YouTube version on

**Comment by Felting** [comment]
> > The old dark theme option doesn't work on video options with nomodernui and fake YouTube version on
> 
> That’s strange. I have this video and I don’t seem to have the issue your having?
> RPReplay_Final1692271713.mov

It shows the oled dark mode on the player for me.

**Comment by Felting** [comment]
> Hey is this the issue you were having? 
> 
> ![IMG_2321](https://github.com/arichorn/uYouPlusExtra/assets/78001398/f4665eb0-fc2c-4185-8a3f-17ff7fee627c)
> 
> 

Yes but with the old layout

**Comment by Felting** [comment]
![image](https://github.com/arichorn/uYouPlusExtra/assets/92234562/3392ce70-599f-4736-ad68-161bfce025c6)

**Comment by RevENGE-code** [comment]
> > Also unrelated, but I can’t turn on notifications on the modded app. Is there a way to make the allow notifications pop up happen again?
> 
> 
> 
> Can't, Sideloaded YouTube is unable to have notification pop ups and I have no clue if there is even a fix for that

This is not true. This is a [bug in uYou](https://github.com/qnblackcat/uYouPlus/issues/978). Sideloaded apps can prompt to enable notifications, but of course push notifications wont auctually work. However any uYou+ version after 'uYouPlus_18.07.5_2.3.1' won't prompt to. `uYouPlus_18.07.5_2.3.1` and the versions before it will.

**Comment by aricloverEXTRA** [comment]
Closed, YTNoModernUI's purpose is gone next release of uYouEnhanced, It got broken overtime in newer YT updates.

Also I'm pretty sure the theme bug is fixed without YTNoModernUI enabled, enjoy?
